Source/Icebreaker/Content/styles.css

```diff
@@ -328,55 +221,16 @@ body {
         outline-offset: -0.25rem
     }
 
```

Comment:
Is there a reason there is no .theme-highContrast .primary-button:active:focus section?

Reply:
@jotrick There is a section if you look at line 216

Reply:
Sorry! Hidden under the fold in GitHub.